frame/benchmarking/pov/src/lib.rs

```diff
@@ -120,14 +120,14 @@ pub mod pallet {
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
 		#[pallet::call_index(0)]
-		#[pallet::weight(0)]
+		#[pallet::weight({0})]
```

Comment:
Note: Some pallets cannot be put in dev mode, since they access the storage info.  
But we could add custom lint attributes to suppress certain warnings, similar to how clippy works.